pkg/webhook/conversion/conversion_test.go

```diff
@@ -200,3 +200,39 @@ var _ = Describe("Conversion Webhook", func() {
 	})
 
 })
+
+var _ = Describe("Convertibility Check", func() {
+
+	var scheme *runtime.Scheme
+
+	BeforeEach(func() {
+
+		scheme = kscheme.Scheme
+		Expect(jobsapis.AddToScheme(scheme)).To(Succeed())
+
+	})
+
+	It("should not return error for convertible types", func() {
+		obj := &jobsv2.ExternalJob{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "ExternalJob",
+				APIVersion: "jobs.example.org/v2",
+			},
+		}
+
+		err := CheckConvertibility(scheme, obj)
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	It("should not return error for a built-in multi-version type", func() {
```

Comment:
we should probably have a test for the error case

Reply:
Yes, we should. It requires adding new type definitions (with partial conversion implementation), I will do that in a follow up PR.